### ISSUE TITLE
Plugin: TypeError if no abc on page

### ISSUE THREAD

**Issue by Nickno95** [paulrosen/abcjs#107]
I noticed when using the abcjs plugin, if the page doesn't have any abc notation on it, the console spits out this error:

```
Uncaught TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at Plugin.plugin.start (abcjs_plugin_3.1.3-min.js:9)
    at HTMLDocument.<anonymous> (abcjs_plugin_3.1.3-min.js:9)
plugin.start @ abcjs_plugin_3.1.3-min.js:9
(anonymous) @ abcjs_plugin_3.1.3-min.js:9
```
Probably related to this: [link](https://stackoverflow.com/questions/23359173/javascript-reduce-an-empty-array)

**Comment by paulrosen** [comment]
Ok, I'll take care of that in the next version.

**Comment by paulrosen** [comment]
Fixed in 3.2.0
